<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="8" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Счётчик</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="CRBComboBox" name="cmbEventPurpose"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="14" column="1">
    <widget class="QCheckBox" name="chkNotAccount">
     <property name="text">
      <string>Не выставлять счет</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="CDateEdit" name="edtEndDate"/>
   </item>
   <item row="6" column="1">
    <widget class="CDateEdit" name="edtBegDate"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblBegDate">
     <property name="text">
      <string>Дата начала</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="CRBComboBox" name="cmbCounter"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edtRegionalCode">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblName">
     <property name="text">
      <string>&amp;Наименование</string>
     </property>
     <property name="buddy">
      <cstring>edtName</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="edtName"/>
   </item>
   <item row="13" column="1">
    <widget class="QCheckBox" name="chkContinued">
     <property name="text">
      <string>&amp;Не закончен</string>
     </property>
    </widget>
   </item>
   <item row="15" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>103</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="16" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblFederalCode">
     <property name="text">
      <string>&amp;Федеральный код</string>
     </property>
     <property name="buddy">
      <cstring>edtFederalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="edtFederalCode"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblRegionalCode">
     <property name="text">
      <string>&amp;Региональный код</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="edtCode">
     <property name="maxLength">
      <number>8</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Цель визита</string>
     </property>
     <property name="buddy">
      <cstring>cmbEventPurpose</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblEndDate">
     <property name="text">
      <string>Дата окончания</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="lblSocStatusType">
     <property name="text">
      <string>Тип</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblSocStatus">
     <property name="text">
      <string>Соц. статус</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="CRBComboBox" name="cmbSocStatusClass"/>
   </item>
   <item row="11" column="0">
    <widget class="QLabel" name="lblSocStatusClass">
     <property name="text">
      <string>Класс</string>
     </property>
    </widget>
   </item>
   <item row="12" column="1">
    <widget class="CRBComboBox" name="cmbSocStatusType"/>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblAttachType">
     <property name="text">
      <string>Тип прикрепления</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="CRBComboBox" name="cmbAttachType"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbEventPurpose</tabstop>
  <tabstop>edtCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>edtName</tabstop>
  <tabstop>edtFederalCode</tabstop>
  <tabstop>cmbCounter</tabstop>
  <tabstop>chkContinued</tabstop>
  <tabstop>chkNotAccount</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>158</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>238</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
